util/find-doc-nits

```diff
@@ -447,6 +447,24 @@ sub check {
         check_section_location($id, $contents, "EXAMPLES", "SEE ALSO");
     }
 
+    # Make sure every link has a section.
+    while ( $contents =~ /$markup_re/msg ) {
+        my $target = $1;
+        next unless $target =~ /L</;            # Skip if not L<...>
+        next if $target =~ /::/;                # Perl module
+        next if $target =~ m@L</@;              # Links within the page
+        next if $target =~ /^L<https?:/;        # URL link
+        next if $target =~ m@\([1357]\)>$@;     # Has a section
+        next if $target =~ m@\([1357]\)/[^>].*>$@; # Has a section/anchor
```

Comment:
`[^>]` is superfluous at this point

Reply:
yeah, when "*" become ".*" I missed that.  fixed.